src/modules/esl-related-target/test/related-target.test.ts

```diff
@@ -0,0 +1,53 @@
+import {ESLRelatedTarget} from '../core/esl-related-target';
+import {ESLToggleable} from '../../esl-toggleable/core/esl-toggleable';
+
+describe('ESLRelatedTarget: show/hide depending on mixin element state', () => {
+  const $el = document.createElement(ESLToggleable.is) as ESLToggleable;
+  const $relatedEl = document.createElement(ESLToggleable.is) as ESLToggleable;
```

Comment:
We have static method to create instance of successor of ESLBaseElement
```suggestion
  const $el = ESLToggleable.create();
  const $relatedEl = ESLToggleable.create();
```